etc/systemd/dnf-automatic-install.timer

```diff
@@ -2,6 +2,8 @@
 Description=dnf-automatic-install timer
 # See comment in dnf-makecache.service
 ConditionPathExists=!/run/ostree-booted
+Wants=network-online.target
```

Comment:
@ignatenkobrain Please could you finish the review, or provide your solution? Otherwise there will be no progress.